kemdk/src/main/java/dk/gls/kemdk/EMDKThrowable.kt

```diff
@@ -0,0 +1,9 @@
+package dk.gls.kemdk
+
+import com.symbol.emdk.EMDKResults
+
+sealed class EMDKThrowable : Throwable() {
+    class ProfileLoadThrowable(val throwable: Throwable) : EMDKThrowable()
+    class ProfileXMLThrowable(val emdkResult: EMDKResults?) : EMDKThrowable()
+    class UnableToRetrieveSerial(val serialResult: String?) : EMDKThrowable()
+}
```

Comment:
Consider adding some java doc.
You could add the nice documentation you have in readme for each case with your potential fixes.